zcash_client_backend/src/fees.rs

```diff
@@ -318,6 +380,24 @@ pub trait ChangeStrategy {
     /// change outputs recommended by this operation. If insufficient funds are available to
     /// supply the requested outputs and required fees, implementations should return
     /// [`ChangeError::InsufficientFunds`].
+    ///
+    /// If the inputs include notes or UTXOs that are not economic to spend in the context
+    /// of this input selection, a [`ChangeError::DustInputs`] error can be returned
+    /// indicating inputs that should be removed from the selection (all of which will
+    /// have value less than or equal to the marginal fee). The caller should order the
+    /// inputs from most to least preferred to spend within each pool, so that the most
+    /// preferred ones are less likely to be indicated to remove.
+    ///
+    #[cfg_attr(
+        feature = "transparent-inputs",
+        doc = "`ephemeral_parameters` can be used to specify variations on how balance
+               and fees are computed that are relevant to transactions using ephemeral
+               transparent outputs; see `EphemeralParameters::new`."
```

Comment:
I tried to correct the intra-doc link error but didn't spot that this has been refactored into `EphemeralBalance`. Please fix this documentation and force-push over the last commit.